driver-core/src/main/com/mongodb/connection/ConnectionPoolSettings.java

```diff
@@ -410,7 +410,7 @@ public String toString() {
     }
 
     ConnectionPoolSettings(final Builder builder) {
-        isTrue("maxSize > 0", builder.maxSize > 0);
+        isTrue("maxSize >= 0", builder.maxSize >= 0);
```

Comment:
Consider adding a unit test in `ConnectionPoolSettingsSpecification`

Reply:
Done.